### PR TITLE
allInOneNotes: Use 2.10 Core

### PR DESCRIPTION
**Pull request by kingthorin**
- Update build file, CHANGELOG, and change code to use updated logging infrastructure.

Part of: zaproxy/zaproxy#6376

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: addOns/allinonenotes/src/main/java/org/zaproxy/zap/extension/allinonenotes/EventConsumerImpl.java [hunk 1]
```diff
@@ -105,8 +106,7 @@ public void eventReceived(Event event) {
                 break;
             case HistoryReferenceEventPublisher.EVENT_REMOVED:
                 // This only gets fired if request is removed
-                LOGGER.debug("NOTE REMOVE EVENT recieved...");
-                LOGGER.debug(event.getParameters().toString());
+                LOGGER.debug("NOTE REMOVE EVENT recieved... {}", event.getParameters().toString());
```
Comment (thc202) [inline addOns/allinonenotes/src/main/java/org/zaproxy/zap/extension/allinonenotes/EventConsumerImpl.java:109]: Could you also fix the typo in received? (In all debug messages.)

Comment (kingthorin) [reply addOns/allinonenotes/src/main/java/org/zaproxy/zap/extension/allinonenotes/EventConsumerImpl.java:109]: Yup good catch

**Comment by kingthorin** [comment]
Done

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by kingthorin** [comment]
Any ideas on the LGTM failure?

**Comment by thc202** [comment]
Failure to download the 2.10-SNAPSHOT, seems they were purged.

**Comment by kingthorin** [comment]
But shouldn't it be using the standard library not the SNAPSHOT? Or is it part of the diffing process that's after the SNAPSHOT?

**Comment by thc202** [comment]
It was caused by other add-on still using the SNAPSHOT.

**Comment by kingthorin** [comment]
Okay, so can we ignore it for now? I see the other similar PR has two approvals and also failed.

**Comment by thc202** [comment]
Yes.

**Review by psiinon — APPROVED** [review]

**Pull request merged**